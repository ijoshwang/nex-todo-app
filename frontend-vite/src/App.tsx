import Todos from './components/Todos'

import './App.css'

function App() {
  return (
    <>
      <Todos></Todos>
      {/* <Checklist></Checklist> */}
      {/* <Todos></Todos> */}
    </>
  )
}

export default App
