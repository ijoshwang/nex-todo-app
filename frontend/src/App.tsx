import Checklist from './Checklist'
import Todos from './Todos'

import './App.css'

function App() {
  return (
    <>
      <Checklist></Checklist>
      <Todos></Todos>
    </>
  )
}

export default App
