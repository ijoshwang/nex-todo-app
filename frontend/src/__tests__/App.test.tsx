import { MemoryRouter } from 'react-router-dom'
import { render } from '@testing-library/react'

import App from '../App'

describe('App Component', () => {
  test('renders the whole app correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    )
    expect(container).toMatchSnapshot()
  })
})
