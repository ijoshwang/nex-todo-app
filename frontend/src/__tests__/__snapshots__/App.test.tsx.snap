// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders the whole app correctly 1`] = `
<div>
  <div
    role="alert"
  >
    <div
      aria-busy="true"
      aria-live="polite"
      class="ant-spin ant-spin-spinning css-dev-only-do-not-override-1uq9j6g"
    >
      <span
        class="ant-spin-dot-holder"
      >
        <span
          class="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            class="ant-spin-dot-item"
          />
          <i
            class="ant-spin-dot-item"
          />
          <i
            class="ant-spin-dot-item"
          />
          <i
            class="ant-spin-dot-item"
          />
        </span>
      </span>
    </div>
  </div>
</div>
`;
